com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -546,40 +551,97 @@ public int additionalLightsShadowmapResolution
             get { return (int)m_AdditionalLightsShadowmapResolution; }
         }
 
+        /// <summary>
+        /// Controls the maximum distance at which shadows are visible.
+        /// </summary>
         public float shadowDistance
         {
             get { return m_ShadowDistance; }
             set { m_ShadowDistance = Mathf.Max(0.0f, value); }
         }
 
+        // MTT this is obsolete. What todo?
         public ShadowCascadesOption shadowCascadeOption
```

Comment:
If I understand correctly we are moving from this to count. Then we should deprecate this and the enum.

Reply:
done